reference/snapshot.md

```diff
@@ -29,7 +29,7 @@ Currently, Mayastor supports the following operations related to volume snapshot
 ## Prerequisites
 
 
-1. Deploy and configure Mayastor by following the steps given [here](quickstart/deploy-mayastor.md) and create disk pools. 
+1. Deploy and configure Mayastor by following the steps given [here](https://mayastor.gitbook.io/introduction/quickstart/deploy-mayastor.md) and create disk pools. 
```

Comment:
Previous PR had used relative path, example: `../` 
If that works it'd be better IMHO

Reply:
I've tried using the relative path but it's not working. So, I've used the absolute path to fix the issue.

Reply:
The relative path change is already live, but not working. Not sure of the reason. Also noticed that elsewhere in the docs, we seem to have used absolute paths (except SUMMARY which is at top-level).